contracts/trusttoken/mocks/MockTrustToken.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {TrustToken} from "../TrustToken.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+
+/**
+ * @title MockTrustToken
+ * @dev The TrustToken contract is a claimable contract where the
+ * owner can only mint or transfer ownership. TrustTokens use 8 decimals
+ * in order to prevent rewards from getting stuck in the remainder on division.
+ * Tolerates dilution to slash stake and accept rewards.
+ */
+// prettier-ignore
```

Comment:
Let's avoid disabling prettier unless it formats code considerably uglier than it could be.